docs/concepts/models.md

```diff
@@ -1122,43 +1122,47 @@ assert error.model_dump() == {
     [`pydantic.main.create_model`][pydantic.main.create_model]<br>
 
 There are some occasions where it is desirable to create a model using runtime information to specify the fields.
-For this Pydantic provides the `create_model` function to allow models to be created on the fly:
+Pydantic provides the [`create_model()`][pydantic.create_model] function to allow models to be created dynamically:
 
 ```python
 from pydantic import BaseModel, create_model
 
-DynamicFoobarModel = create_model(
-    'DynamicFoobarModel', foo=(str, ...), bar=(int, 123)
-)
+DynamicFoobarModel = create_model('DynamicFoobarModel', foo=str, bar=(int, 123))
+
+# Equivalent to:
 
 
 class StaticFoobarModel(BaseModel):
     foo: str
     bar: int = 123
 ```
 
-Here `StaticFoobarModel` and `DynamicFoobarModel` are identical.
-
 Fields are defined by one of the following tuple forms:
+Field definitions are specified as keyword arguments, and should either be:
 
-* `(<type>, <default value>)`
-* `(<type>, Field(...))`
-* `typing.Annotated[<type>, Field(...)]`
+- A single element, representing the type annotation of the field.
+- A two-tuple, the first element being the type and the second element the assigned value
+  (either a default or the [`Field()`][pydantic.Field] function).
```

Comment:
I generally hesitate to admit that `Field()` is a function, we sort of sell it more like a dataclass. Perhaps we could phrase this differently, as to not confuse folks? Same thing later on in this PR as well

Reply:
I think it's best to keep it this way, because the [API docs](https://docs.pydantic.dev/latest/api/fields/#pydantic.fields.Field) shows it as a function anyway, and the `dataclasses` module [documents `field`](https://docs.python.org/3/library/dataclasses.html#dataclasses.field) as being a function